src/brogue/MainMenu.c

```diff
@@ -842,7 +842,10 @@ void mainBrogueJunction() {
                             rogue.playbackPaused = false;
                             pausePlayback();
                         }
-
+#ifdef ENABLE_PLAYBACK_SWITCH
+                        if(rogue.gameHasEnded) // We are coming from the end of a recording the user has taken
+                            break;             // over. No more event checks, that has already been handled
```

Comment:
Definitely do not like wrapped adjacent comments, far too difficult to edit either code or comment.